backend/db/migrate/20170801101708_create_clients.rb

```diff
@@ -0,0 +1,9 @@
+class CreateClients < ActiveRecord::Migration[5.1]
+  def change
+    create_table :clients do |t|
+      t.string :token
+
```

Comment:
Is it possible to have the `name` here of the client? I think we would like to know if which client is doing the request and which token are associated to them. @ninariccimarie